arangod/Cluster/HeartbeatThread.cpp

```diff
@@ -285,7 +285,7 @@ void HeartbeatThread::runDBServer() {
     if (!registered) {
       LOG_TOPIC(ERR, Logger::HEARTBEAT)
           << "Couldn't register plan change in agency!";
-      std::this_thread::sleep_for(std::chrono::seconds(1));
+      std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
Just curious.  Why did you change from std::chrono::seconds(1) to std::chrono::milliseconds(1000)?

Reply:
just cause i tried with different times. that's it really. let me go back